src/cli/global/expose.rs

```diff
@@ -0,0 +1,83 @@
+use std::{error::Error, path::PathBuf};
+
+use clap::Parser;
+use itertools::Itertools;
+use miette::IntoDiagnostic;
+use pixi_config::ConfigCli;
+use rattler_shell::shell::ShellEnum;
+
+use crate::global::{expose_add, expose_remove};
+
+use crate::{
+    global::{
+        self, create_executable_scripts, script_exec_mapping, EnvDir, EnvironmentName, ExposedKey,
+    },
+    prefix::{create_activation_script, Prefix},
+};
+
+#[derive(Parser, Debug)]
+pub struct AddArgs {
+    /// The binary to add as executable in the form of key=value (e.g. python=python3.10)
+    #[arg(value_parser = parse_key_val)]
+    name: Vec<(String, String)>,
+
+    #[clap(long)]
+    environment: String,
```

Comment:
```suggestion
    #[clap(long, short)]
    env: String,
```

As named in the proposal, I forgot to mention the short form `-e` in the proposal, but we should add it to be consistent with different pixi commands